DTAConfig/HotkeyConfigurationWindow.Hotkey.cs

```diff
@@ -0,0 +1,268 @@
+                Keys.VolumeUp => throw new NotImplementedException(),
+                Keys.MediaNextTrack => throw new NotImplementedException(),
+                Keys.MediaPreviousTrack => throw new NotImplementedException(),
+                Keys.MediaStop => throw new NotImplementedException(),
+                Keys.MediaPlayPause => throw new NotImplementedException(),
+                Keys.LaunchMail => throw new NotImplementedException(),
+                Keys.SelectMedia => throw new NotImplementedException(),
+                Keys.LaunchApplication1 => throw new NotImplementedException(),
+                Keys.LaunchApplication2 => throw new NotImplementedException(),
+                Keys.OemSemicolon => throw new NotImplementedException(),
+                Keys.OemPlus => throw new NotImplementedException(),
+                Keys.OemComma => throw new NotImplementedException(),
+                Keys.OemMinus => throw new NotImplementedException(),
+                Keys.OemPeriod => throw new NotImplementedException(),
+                Keys.OemQuestion => throw new NotImplementedException(),
+                Keys.OemTilde => throw new NotImplementedException(),
+                Keys.OemOpenBrackets => throw new NotImplementedException(),
+                Keys.OemPipe => throw new NotImplementedException(),
+                Keys.OemCloseBrackets => throw new NotImplementedException(),
+                Keys.OemQuotes => throw new NotImplementedException(),
+                Keys.Oem8 => throw new NotImplementedException(),
+                Keys.OemBackslash => throw new NotImplementedException(),
+                Keys.ProcessKey => throw new NotImplementedException(),
+                Keys.Attn => throw new NotImplementedException(),
+                Keys.Crsel => throw new NotImplementedException(),
+                Keys.Exsel => throw new NotImplementedException(),
+                Keys.EraseEof => throw new NotImplementedException(),
+                Keys.Play => throw new NotImplementedException(),
+                Keys.Zoom => throw new NotImplementedException(),
+                Keys.Pa1 => throw new NotImplementedException(),
+                Keys.OemClear => throw new NotImplementedException(),
+                Keys.ChatPadGreen => throw new NotImplementedException(),
+                Keys.ChatPadOrange => throw new NotImplementedException(),
+                Keys.Pause => throw new NotImplementedException(),
+                Keys.ImeConvert => throw new NotImplementedException(),
+                Keys.ImeNoConvert => throw new NotImplementedException(),
+                Keys.Kana => throw new NotImplementedException(),
+                Keys.Kanji => throw new NotImplementedException(),
+                Keys.OemAuto => throw new NotImplementedException(),
+                Keys.OemCopy => throw new NotImplementedException(),
+                Keys.OemEnlW => throw new NotImplementedException(),
```

Comment:
I can't see any point in listing all of them here, since this is just an override for specific key names. I think this'll also just crash the client.

Reply:
This is probably the content that was automatically fixed by `dotnet format`.
There is an add missing case in the parser.